src/partialRight.js

```diff
@@ -26,4 +30,37 @@ var flip = require('./flip');
  *
  *      greetMsJaneJones('Hello'); //=> 'Hello, Ms. Jane Jones!'
  */
-module.exports = _createPartialApplicator(flip(_concat));
+module.exports = _curry2(function partialRight(fn, partialArgs) {
```

Comment:
Is it perhaps worth simplifying the implementation of `partialRight` by deriving it from `partial`?

Something like:
```js
function partialRight(fn, partialArgs) {
    var leftArgs = [];
    var leftN = fn.length - partialArgs.length;
    while (leftN > 0) {
        leftArgs.push(__);
        leftN = leftN - 1;
    }
    return partial(fn, leftArgs.concat(partialArgs));
}
```